src/common/config_opts.h

```diff
@@ -13,7 +13,7 @@
  */
 
 /* note: no header guard */
-OPTION(host, OPT_STR, "localhost")
+OPTION(host, OPT_STR, "default")  // "default" means that host will be `hostname -s` 
```

Comment:
hrm, maybe what we should really do is eliminate 'host' as a config option and turn it into a proper meta.  i'm not sure why users would be configuring this?